sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/RelationshipSamples.cs

```diff
@@ -21,36 +21,57 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
             // Despite not being a good code practice, this prevents code snippets from being out of context for the user when making API calls that accept Ids as parameters.
 
             PrintHeader("RELATIONSHIP SAMPLE");
+            string sampleBuildingModelId = "dtmi:com:samples:SampleBuilding;1";
+            string sampleFloorModelId = "dtmi:com:samples:SampleFloor;1";
+            try
+            {
+                Console.WriteLine($"Deleting model Id '{sampleBuildingModelId}' if it exists.");
```

Comment:
If the sample runs once but fails to cleanup, we need to make sure that these model Ids are available to create upon any subsequent runs